pvlib/modelchain.py

```diff
@@ -260,6 +263,27 @@ def get_orientation(strategy, **kwargs):
     return surface_tilt, surface_azimuth
 
 
+@dataclass
+class ModelChainResult:
+    T = TypeVar('T')
+    PerArray = Union[T, Tuple[T, ...]]
+    # system-level information
+    solar_position: pd.DataFrame = field(default=None)
+    airmass: pd.DataFrame = field(default=None)
+    ac: pd.Series = field(default=None)
```

Comment:
why doesn't the "system-level information" use `Optional`? My understanding is that `Optional` serves to cover the `None` type that exists until that particular result is calculated. So I think it would apply to these parameters too.

Reply:
That was a mistake, thanks for catching it.